### ISSUE TITLE
EPG view timeline not scrolling in TV

### ISSUE THREAD

**Issue by manjusha221** [Duna/TV-EPG-Android-Recyclerview#12]
Hi Duna,

I tried running the epgview in tv, timeline is not scrolling with item move

**Comment by Duna** [comment]
Yes you have right, remote tv focus needs to be handled individually for each cell
Can you help with a pull request?
